_includes/contact_form.html

```diff
@@ -350,13 +350,14 @@
 
 <form
   id="contact-us-form"
-  action="https://webto.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8"
+  action="{{ site.contact_form_action }}"
   method="POST"
   onSubmit="return remapvalues(this); return true;"
   class="usa-form"
+  data-captcha-enabled="{{ site.contact_form_captcha_enabled }}"
```

Comment:
I recall in last tinkering with this, Salesforce has an option to enable a "debug" mode:

>To test the Web-to-Case form, add the line `<input type="hidden" name="debug" value="1">` to the code. This line redirects you to a debugging page when you submit the form. Don't forget to remove it before releasing the Web-to-Case page to your website.

https://help.salesforce.com/articleView?id=sf.customizesupport_web_to_case_enable_recaptcha.htm&type=5

Maybe as an alternative to this handling, we could just include this input field based on the configuration value?

Reply:
Love that idea, added in fce00c0b2849c9b4689c953cdcd1c7b6a13094d0